br/cmd/tidb-lightning/main.go

```diff
@@ -22,12 +22,13 @@ import (
 	"runtime/debug"
 	"syscall"
 
+	"go.uber.org/zap"
+
```

Comment:
remove this empty line.

Reply:
https://github.com/pingcap/tidb/pull/33303/commits/5ac26f09ba5888a07b2345d12451172bc8a5c6b5